src/main/kotlin/org/b12x/gfe/utilities/DirectoryManagement.kt

```diff
@@ -0,0 +1,92 @@
+package org.b12x.gfe.utilities
+
+import java.io.File
+import java.nio.file.Files.createDirectories
+import java.nio.file.Paths
+
+class DirectoryManagement {
+
+    /**
+     * Checks for the existence of a folder.
+     *
+     * @param pathInQuestion the path to the folder to check
+     * @return whether or not the folder exists
+     */
+    fun doesFolderExist(pathInQuestion: String): Boolean {
+        return File(pathInQuestion).exists()
+    }
+
+    /**
+     * Checks to see if there's anything in the folder.
+     *
+     * @param pathInQuestion the path to the folder to check
+     * @return if the folder contains anything.
+     */
+    fun doesFolderContainFiles(pathInQuestion: String): Boolean {
+        val files = File(pathInQuestion).list()
```

Comment:
Will this code fail if it is passed a nonexistent path? If so, that should be specified in the API.

In any case, it seems you are calling the `File` constructor a lot. Maybe some of these methods should be extension functions. Let me know if you'd like to discuss.